spells.txt

```diff
@@ -27,11 +27,13 @@
 "advanced automata repair"
 {
   symbol = "🔧"
-  effect = Area 3 Filter AUTOMATON RegrowBodyPart 10
+  #GD: better healing / should make a main pull to keeperrl to add Automation as a flesh type.
+  effect = Area 3 Filter AUTOMATON Filter ALLY Chain{ RegrowBodyPart 10 Heal FLESH }
```

Comment:
automata repair had no Heal FLESH part, so I do not think this one should neither

Reply:
Fixed